.circleci/check_mysql_connection.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+[ -n "$MYSQL_USER" ] || MYSQL_USER=test
+[ -n "$MYSQL_PASSWORD" ] || MYSQL_PASSWORD=test
```

Comment:
Bash has a cute trick in its sleeve:

```bash
MYSQL_USER="${MYSQL_USER:-test}"
```

which is, in my experience, more conventional for default parameter values.